style.css

```diff
@@ -0,0 +1,66 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500&family=Poppins:wght@500&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px;
+  align-items: center;
+}
+
+.nav-bar a {
+  text-decoration: none;
+  line-height: 20px;
+  font-size: 18px;
+}
+
+.main-section {
```

Comment:
Using `flexbox` it's really useful to make your website responsive, as this is part of the requirements too.

- Kindly add a `display: flex;` to your main section, this will help you on your desktop version to make it responsive also in higher resolutions 🥇 

Here's a guide for [Flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) if you need some help understanding how it's works better